todo_app/app.py

```diff
@@ -1,28 +1,61 @@
+from board_item import Board_item
 from flask import Flask, render_template, redirect, url_for, request
 
 from todo_app.flask_config import Config
 from todo_app.data import session_items as session
 
+import requests
+import os
+from dotenv import load_dotenv
+import json
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
+load_dotenv()
```

Comment:
You shouldn't need this line explicitly when running the app with `poetry run flask run` - the python-dotenv module will work with flask to do so by default